src/player.ts

```diff
@@ -405,4 +405,41 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
+   * Sets the playback speed of the player. Fast forward, slow motion and reverse playback are supported.
+   * @note
+   * - Slow motion is used by values between `0` and `1`.
+   * - Fast forward by values greater than `2`.
```

Comment:
What happens between 1 and 2?  on Android, all values above 1 are considered fast forward. But I am not sure if there is some misalignment as to what fast forward means. 🤔

Reply:
Yes, it make sense to change it to `1`. For some reason it was `2` on iOS, but we decided to go for `1` there too.  
Address in 16c8becb9c43d19f6943687349db24730862c79b